docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,189 @@
+
+
+<span class="notes">**Note:** You'll still need Mbed CLI for older versions of Mbed OS (6.x and older). You can install both tools side by side.</span>
+
+# Install or upgrade
+
+Mbed Tools is a Python package called `mbed-tools`, so you can install it with pip.
+<!--other than being a super vague name, it's a problem that it's a single tool with a plural name. "Mbed Tools is"  just parses as an error!-->
+
+<span class="tips">**Tip:** We recommend using a virtual environment to avoid any Python dependency conflicts. We recommend the [Pipenv](https://github.com/pypa/pipenv/blob/master/README.md) tool to manage your virtual development environment.
+
+## Prerequisite
+
+- Python 3.6 or newer. Install for [Windows](https://docs.python.org/3/using/windows.html), [Linux](https://docs.python.org/3/using/unix.html) or [macOS](https://docs.python.org/3/using/mac.html).
+- Pip (if not included in your Python installation). [Install for all operating systems](https://pip.pypa.io/en/stable/installing/).
+- CMake. [Install version 3.18.1 or newer for all operating systems](https://cmake.org/install/).
+- Ninja [Install version 1.0 or newer for all operating systems](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages).
+- One of the support toolchains listed [in the build tools overview](../build-tools/index.html).
+
+## Install
+
+Use pip to install:
+
+- To install the latest release:
+
+    ```
+    python -m pip install mbed-tools
+    ```
+
+- To install a specific version:
+
+    ```
+    python -m pip install mbed-tools==<version number in major.minor.patch format>
+    ```
+
+- To install a pre-release or development version:
+
+    ```
+    python -m pip install mbed-tools --pre
+    ```
+
+## Upgrade
+
+Use pip to upgrade your version:
+
+```
+python -m pip install mbed-tools --upgrade
+```
+
+# Use
+
+Currently, the new Mbed Tools supports three boards: `K64F`, `DISCO_L475VG_IOT01A` and `NRF52840_DK`.
+
+For help:
+
+- To get help for all commands, use:
+
+    ```
+    mbed-tools --help
+    ```
+
+- To get help for a specific command, use `mbed-tools <command> --help`. For example, for helping with listing connected devices (the `devices` command):
+
+    ```
+    mbed-tools devices --help
+    ```
+
+## Create a project
+
+You can create a new project or create a local copy of one of our example applications.
+
+### Create a new project
+
+To create a new Mbed OS project in a specified path:
+
+- To create the project and download a new copy of Mbed OS (latest official release):
+
+    ```
+    mbed-tools init <PATH>
+    ```
+
+    The path can be:
+    - Absolute. `init` will create the folder if it doesn't exist.
+    - Relative.
+
+    If you have already created a project folder, you can use `.`
+
+    If you want the `init` command to create a project folder, use `.\<folder-name>`.
+
```

Comment:
@iriark01 We should also document `-c` option which allows sym-linking existing mbd-os. Please check https://github.com/ARMmbed/mbed-tools/issues/65

Reply:
I removed that because it wasn't fully functional

Reply:
@iriark01 Did you face any errors when you ran`mbed-tools init -c`?

Reply:
I was simply told it doesn't work yet

Reply:
This has been added recently, please review and resolve this comment @urutva

Reply:
@iriark01 @jainvikas8 I've tried `mbed-tools init -c` after commenting and it works as expected. It should be added to the document.

Reply:
It's in line 42 in the file about using it